lib/util/template/index.spec.ts

```diff
@@ -48,6 +48,22 @@ describe('util/template/index', () => {
     expect(output).toContain('True');
   });
 
+  it('ifChanged - yes', () => {
+    const userTemplate =
+      '{{#ifChanged depName newName}}{{{newName}}}{{/ifChanged}}';
```

Comment:
Why not simply
```suggestion
      '{{#unless (equals depName newName)}}{{{newName}}}{{/unless}}';
```
Not sure it is worth to add a new helper here

Reply:
Thanks! I wasn't aware of this helper

Reply:
I added some tests locally that failed. I will try again to be sure.

Reply:
Updated template with an if to check for the undefined `newName`

New repo runs:
No replacement pr: https://github.com/Rahul-renovate-testing/replacements-table-2/pull/3
Replacement pr: https://github.com/Rahul-renovate-testing/replacements-table-2/pull/1